src/libsync/configfile.cpp

```diff
@@ -557,7 +557,7 @@ chrono::milliseconds ConfigFile::notificationRefreshInterval(const QString &conn
     QSettings settings(configFile(), QSettings::IniFormat);
     settings.beginGroup(con);
 
-    auto defaultInterval = chrono::minutes(5);
+    auto defaultInterval = chrono::minutes(1);
```

Comment:
@claucambra @mgallien I am not sure we need to change this for every user. Won't we end up creating a high load on the server in case of thousands of clients refresh notifications every minute? AFAIK, push notifications will almost always be available, so, we can assume the interval of 5 minutes is fine for the rest of the cases, but, if not, a user can change it directly in the custom config.

Reply:
Yes, this is true... on the flip side, though, the only connected server I have on my client that supports push notifications is our work instance, and in those conditions this means if we leave it at 5 minutes the likelihood of the call notifications being received in a timely way by the client get really close to 0 :/

Reply:
I am no expert here, but did we not set the default to 5 min only for cases there is a push server? Did we change it for all desktop clients? Can we not set it to 30 seconds without the HPB and 5 minutes with the HPB?

If you then have load issues from the clients, use the HPB.

Reply:
I agree with @jospoortvliet

Reply:
@jospoortvliet @mgallien At least in the user model, which handles notifications, this is not the case -- when push notifications are connected to the server, the polling is stopped altogether and notifications are fetched solely through the push notification method. If the connection is lost or the push notifications are no longer available, the polling is re-initiated

Reply:
Relevant code:https://github.com/nextcloud/desktop/blob/14220e46cd316083bb4e36069aba2db44bbb821d/src/gui/tray/usermodel.cpp#L152